### DISCUSSION TITLE
Live demo

### DISCUSSION THREAD

**Discussion by trusktr** [udevbe/greenfield#148] [General]
Is there a live demo of this running somewhere?

**Comment by Zubnix** [comment]
Not for the moment. I guess it does make sense to host the compositor-shell somewhere as it's fairly simple and doesn't require any login 👍🏻

**Reply by Zubnix** [reply]
now available here https://desktop.greenfield.app/

Make sure to add the correct CORS header when starting your compositor-proxy e.g.:  `yarn start --allow-origin=https://desktop.greenfield.app`
